### PR TITLE
[DNM] *: tiny cleanup

### PR DESCRIPTION
**Pull request by BusyJay**
- disable snappy compression
- add some perf metrics

### CODE REVIEW

File: src/raftstore/store/store.rs [hunk 1]
```diff
@@ -709,16 +711,19 @@ impl<T: Transport, C: PdClient> mio::Handler for Store<T, C> {
     type Message = Msg;
 
     fn notify(&mut self, event_loop: &mut EventLoop<Self>, msg: Msg) {
+        let t = Instant::now();
         match msg {
             Msg::RaftMessage(data) => {
                 if let Err(e) = self.on_raft_message(data) {
                     error!("handle raft message err: {:?}", e);
                 }
+                debug!("handle raftmessage takes {:?}", t.elapsed());
```
Comment (siddontang) [inline src/raftstore/store/store.rs]: why not put all below like timeout? 
I think output Quit cost is ok too.

Comment (BusyJay) [inline src/raftstore/store/store.rs]: Because I want to know what command is being executed.

Comment (siddontang) [inline src/raftstore/store/store.rs]: Msg supports `fmt`, I think can use it directly.

File: src/bin/tikv-server.rs [hunk 2]
```diff
@@ -68,9 +68,12 @@ fn build_raftkv(matches: &Matches,
     let trans = Arc::new(RwLock::new(ServerTransport::new(cluster_id, ch, pd_client.clone())));
 
     let path = get_store_path(matches);
-    let mut opts = rocksdb::Options::new();
-    let mut block_base_opts = rocksdb::BlockBasedOptions::new();
+    let mut opts = RocksdbOptions::new();
+    let mut block_base_opts = BlockBasedOptions::new();
     block_base_opts.set_block_size(64 * 1024);
+    unsafe {
+        ffi::rocksdb_options_set_compression(opts.inner, 0);
```
Comment (ngaut) [inline src/bin/tikv-server.rs]: Could you add an method rocksdb wrapper ?

Comment (siddontang) [inline src/bin/tikv-server.rs]: we can add this in rocksdb lib first.

**Comment by ngaut** [comment]
LGTM

**Pull request merged**